bindings/generator/templates/binding_electron_meths.rs.j2

```diff
@@ -138,12 +138,18 @@ JsFunction
 {
     {% if type.custom_from_rs_bytes -%}
     let custom_from_rs_bytes = {{ type.custom_from_rs_bytes }};
+    // TODO: Some references don't need mutability
+    #[allow(clippy::unnecessary_mut_passed)]
     match custom_from_rs_bytes({{ js_val }}.as_slice({{ mut_cx_ref }}))
     {%- else -%}
+    // TODO: Some references don't need mutability
+    #[allow(clippy::unnecessary_mut_passed)]
     match {{ js_val }}.as_slice({{ mut_cx_ref }}).try_into()
     {%- endif %}
     {
         Ok(val) => val,
+        // TODO: err can't infer type in some case
```

Comment:
Are you sure of the comment ?
I'd say the issue is something like: `format!("{}", err)` is useless in the case `err` is already a `String`
(I'd guess it's a bit more complex: `throw_type_error` want't a trait X, `format!()` returns a `String` that impl trait X, but this is useless if `err` already impl trait X)

Reply:
The problem was an error (CryptoError) which can't infer type because of the previous `.try_into` which is needed for `.to_string`